doc/en/components/layouts/dock-manager-customization.md

```diff
@@ -0,0 +1,90 @@
+---
+title: {Platform} Dock Manager | Customization | Infragistics
+_description: Use Infragistics' {Platform} dock manager component to manage the layout through panes, with the ability to customize it. Check out {ProductName} dock manager tutorials!
+_keywords: dock manager, layout, customization, {ProductName}
+mentionedTypes: ['DockManager']
+---
+
+### Customizing {Platform} Dock Manager
+
+The Infragistics {Platform} Dock Manager component provides you with the properties needed to further customize the layout to suit your specific application requirements.
+Let's dive in and explore how {Platform} DockManager empowers you to create exceptional user interfaces and enhance the productivity of your applications!
+
+<div class="divider--half"></div>
+
+## Focus Panes Programmatically
+
+The `focusPane` method allows developers to dynamically and programmatically focus a specific pane within the layout by providing the `contentId` of the desired pane.
+
+```ts
+this.dockManager.focusPane('content1');
+```
+
+When using the `focusPane` method, the behavior varies depending on the state of the targeted pane. Here's how it works:
+
+1. **Floating Panes**: If the pane is floating, invoking `focusPane` will bring it into focus, set it as the activePane and ensure it appears on top of any other floating panes.
+
+2. **Unpinned Panes**: When the targeted pane is unpinned, `focusPane` will flyout the pane to its open state.
+
+3. **Regular Pinned Panes**: For regular pinned panes, `focusPane` will set the pane as the `activePane`.
+
+
+By utilizing this method, developers can effortlessly control the visibility and positioning of panes based on user interaction or application events.
+
+Try it for yourself in the sample below:
+
+### Demo
```

Comment:
We could skip the 'Demo' subtitles as they are repeating in the left-side navigation menu. In the other dock manager topics we are displaying samples without any subtitle.